batcher.go

```diff
@@ -288,6 +308,40 @@ func (r *Batcher) incTarget(val int) {
 	} // else is val=0, do nothing
 }
 
+func (r *Batcher) tryReserveBatchSlot() bool {
+	if r.maxConcurrentBatches == 0 {
```

Comment:
This seems like it should be documented, if setting this config value to zero disables the limiter entirely.

Reply:
I updated the documentation.